player/src/main/java/com/focusit/jsflight/player/webdriver/SeleniumDriver.java

```diff
@@ -226,68 +286,59 @@ public WebDriver getDriverForEvent(JSONObject event, BrowserType browserType, St
             Integer proxyPort)
     {
         String tabUuid = event.getString(EventConstants.TAB_UUID);
+        WebDriver driver = tabUuidDrivers.get(tabUuid);
 
-        try
+        if (driver == null)
         {
-            WebDriver driver = tabUuidDrivers.get(tabUuid);
-
-            if (driver == null)
+            DesiredCapabilities cap = new DesiredCapabilities();
+            if (!StringUtils.isBlank(proxyHost))
             {
-                DesiredCapabilities cap = new DesiredCapabilities();
-                if (!StringUtils.isBlank(proxyHost))
+                String host = proxyHost;
+                if (proxyPort != 0)
                 {
-                    String host = proxyHost;
-                    if (proxyPort != 0)
-                    {
-                        host += ":" + proxyPort;
-                    }
-                    Proxy proxy = new Proxy();
-                    proxy.setHttpProxy(host).setFtpProxy(host).setSslProxy(host);
-                    cap.setCapability(CapabilityType.PROXY, proxy);
+                    host += ":" + proxyPort;
                 }
+                Proxy proxy = new Proxy();
```

Comment:
Well, I haven't changed anything here except delete try / catch. I agree with you. I'll fix it